src/smc-webapp/r_misc.cjsx

```diff
@@ -1159,9 +1160,14 @@ EditorFileInfoDropdown = rclass
             @render_menu_item(name, icon)
 
     render : ->
-        <DropdownButton style={marginRight:'2px'} id='file_info_button' bsStyle='info' title={<Icon name='info-circle' />} className='pull-left'>
-            {@render_menu_items()}
-        </DropdownButton>
+        <div>
```

Comment:
This code duplication is clearly bad and in no way necessary.  Just make a function `render_dropdown_button`, which takes as input bsSize and className, and call it twice in succession. Then there is no *code* duplication, but we still get two things in the DOM eventually, and CSS works fine... and we can still maintain this code.

I'm serious about bouncing this, because it's important to get basic easy things like this right, so it's possible to do more complicated programming later.